### PR TITLE
(PC-6770 & PC-6154) Notification settings page

### PR DESCRIPTION
**Pull request by blixit**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-6770
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-6154

## Checklist

I have:

- [x] Made sure the title of my PR follows the convention `[PC-XXX] <summary>`.
- [-] Made sure my feature is working on the relevant real / virtual devices.
- [x] Written **unit tests** for my feature.
- [-] Added new reusable components to the AppComponents page and communicate to the team on slack
- [x] Added a **screenshot** for UI tickets.
- [-] Attached a **ticket number** for any added TODO/FIXME \
       (for tech tasks, give the best context about the TODO resolution: what/who/when).

## Screenshots

| \*iOS - [Phone name] | \*Android - [Phone name] | Tablet - [Phone name] | Little - [Phone name] |
| -------------------- | :----------------------: | --------------------: | --------------------: |
|                      | image |                       |                       |

## Deploy hard

If native code (ios/android) was modified, **after** the PR is merged, on the master branch, upgrade the **app version** (+1 patch):

- if you want an hard deployment of the testing environment, use `yarn trigger:testing:deploy`

### CODE REVIEW

**Review by justinemignot — COMMENTED** [review]
Mets à jour messages.po et c'est bon :)

File: src/features/profile/components/reusables.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components/native'
+
+import { ColorsEnum, getSpacing } from 'ui/theme'
+
+export const ProfileContainer = styled.View({
```
Comment (AnoukHello) [inline src/features/profile/components/reusables.tsx:5]: Niveau naming j'ai du mal à voir ce que ça représente :thinking:

Comment (kopax-polyconseil) [reply src/features/profile/components/reusables.tsx:5]: Sans regarder le code, je dirait que c'est un wrapper pour le profile.

**Review by kopax-polyconseil — COMMENTED** [review]
:ok_hand:

**Pull request closed without merging**